app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -247,11 +258,9 @@ export default class AMRHeatmapView extends React.Component {
       <ViewHeader className={cs.viewHeader}>
         <ViewHeader.Content>
           <ViewHeader.Pretitle>
-            Antimicrobial Resistance Heatmap
+            {`Comparing ${sampleIds ? sampleIds.length : ""} Samples`}
```

Comment:
Curious about this change since the other heatmap has the two labels flipped (Comparing X samples is the title). Was this based on a design discussion? Should we flip the one in SamplesHeatmapHeader as well?

Reply:
Yes, this was a request from Jenn @happyimadesignr . I think she would like to flip the other one as well, but I would double check with her.

Reply:
Yes! Would love to switch it on both heatmaps, the large text across the product should be the title of the page. 

A later thing: I'm also trying to make the small text into breadcrumb across the board too, like we're doing for pipeline viz:
<img width="502" alt="Screen Shot 2019-08-08 at 5 05 10 PM" src="https://user-images.githubusercontent.com/8314555/62745278-0dc00d80-b9ff-11e9-8354-c5ca6abffb14.png">